src/mmw/js/src/modeling/models.js

```diff
@@ -751,6 +758,7 @@ var ScenariosCollection = Backbone.Collection.extend({
             aoi_census: aoi_census
         });
 
+        scenario.set('waitToRefresh', true);
```

Comment:
Since this is set to `true` when a new scenario is created, and is never set to `false`, couldn't we simply remove it and all the corresponding `if` conditions?